lib/ui/widgets/web_link_card.dart

```diff
@@ -57,11 +62,24 @@ class WebLinkCard extends StatelessWidget {
   }
 
   /// used to open a website or the security view
-  void _onLinkClicked(String link) {
+  Future<void> _onLinkClicked(String link, BuildContext context) async {
     if (link == 'security') {
       _navigationService.pushNamed(RouterPaths.security);
     } else {
-      _launchInBrowser(link);
+      try {
+        await _launchInBrowser(link);
+      } catch(e) {        
+        final PackageInfo packageInfo = await PackageInfo.fromPlatform();
+
+        final errorMessage = "**Error message : ${e.toString()}\n**Device Infos** \n"
+                "- **Version:** ${packageInfo.version} \n"
+                "- **Build number:** ${packageInfo.buildNumber} \n"
+                "- **Platform operating system:** ${Platform.operatingSystem} \n"
+                "- **Platform operating system version:** ${Platform.operatingSystemVersion} \n";
+
+        _analyticsService.logError("web_link_card", errorMessage);
+        _navigationService.pushNamed(RouterPaths.webView, arguments: _links);
+      }
     }
   }
 
```

Comment:
one last change needed, set the good ets_red_color for ChromeSafariBrowser instead of red